spaceship/spec/tunes/build_spec.rb

```diff
@@ -7,10 +7,6 @@
   describe "properly parses the build from the train" do
     let(:app) { Spaceship::Application.all.first }
 
-    it "inspect works" do
-      expect(Spaceship::Application.all.first.build_trains.values.first.builds.first.inspect).to include("Tunes::Build")
```

Comment:
This seems to me like it is testing the output of `build_trains` rather than the `Tunes::Build` object itself. We are now returning `TestFlight::Build` objects from `build_trains` so this is not longer relevant.